utils/src/test/kotlin/ArchiveUtilsTest.kt

```diff
@@ -143,12 +146,101 @@ class ArchiveUtilsTest : StringSpec() {
 
                     listOf("data.tar.xz", "control.tar.xz").forEach { tarFileName ->
                         val tarFile = outputDir.resolve(tarFileName)
-                        tarFile shouldBe aFile()
+                        tarFile shouldNotBe aFile()
                     }
                 }
             } finally {
                 tempDir.safeDeleteRecursively(force = true)
             }
         }
+
+        "Tar archive can be unpacked with a filter" {
```

Comment:
Can we please group non-filter / filter tests by start using a `WordSpec` like

```
        "Tar archives" should {
            "unpack" {}
            "unpack with a filter" {}
        }
```

Reply:
Converted the test class to a `WordSpec`.

Reply:
Just as a remark: I was meaning to do the conversion to a `WordSpec` in the last commit when introducing the new tests, as before that the conversion does not really have any value, and interferes with the other changes. No need to change that now again, though.